husky_directory/templates/summary_results_table/base.html

```diff
@@ -1,14 +1,12 @@
 <table class="table table-bordered table-striped table-condensed"
        summary="results" width="100%">
-    <thead>
+    <tbody>
```

Comment:
I explicitly changed this to use html best practices. Please do not change it back. If this affects shading, I think the CSS should be updated.

Reply:
Darn it. Well, the old site is wrong then (not an excuse, just...darn you old site).
I'll get into the CSS

Reply:
Not _wrong_, per se. It's just a low-hanging fruit best practices improvement. Since I'm the one who changed it I'm happy to look into updating the CSS since it caused an issue.

Reply:
I started down the CSS route before this code change and went...wait, they left off the `<thead>` section and that makes it work, but not the right way. Took a gamble to see what y'all would say. If I can't get it in a reasonable amount of time, I'll give a shout.

Reply:
okee doke, np. if you feel strongly about jsut putting it back to body, i can relent. this isn't a hill to die on.